clippy_lints/src/needless_pass_by_value.rs

```diff
@@ -22,13 +23,20 @@ use std::borrow::Cow;
 /// sometimes avoid
 /// unnecessary allocations.
 ///
-/// **Known problems:** Hopefully none.
+/// **Known problems:**
+/// * This lint suggests taking an argument by reference,
+/// however sometimes it is better to let users decide the argument type
+/// (by using `Borrow` trait, for example), depending on how the function is used.
```

Comment:
If we suggest this action, we should probably also generate a second structured suggestion that adds the generic argument

Reply:
Hmm I just wanted to note it somewhere as an alternative to taking by reference.